.github/workflows/chromatic.yml

```
@@ -15,33 +14,32 @@ jobs:
     name: Chromatic
     runs-on: ubuntu-latest
     steps:
-      - name: Get Yarn cache path
-        id: yarn-cache
-        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
-
-      - name: Checkout Repo
+      - name: Checkout
         uses: actions/checkout@v3
         with:
-          fetch-depth: 0
+          fetch-depth: 2
 
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
           node-version: 18.17.1
```

```suggestion
          node-version-file: .nvmrc
```
이렇게도 될 것 같아요(https://github.com/actions/setup-node)

---

여기서 `0` 으로 안하면 chromatic에서 에러가 발생해서 (모든 commit 이력을 가져와야함) fetch-depth 부분은 롤백합니다.